0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -246,6 +228,24 @@ This header may be used for source routing.
 
 ## Discussion
 
+### Holds and Payment Reliability
+
+Interledger payments may be transmitted through connectors and intermediary ledgers that are not trusted by the sender or receiver. If connectors fail to pass on payments, money could be lost.
+
+Ledgers MAY provide transfer hold functionality to protect hosts from the risk posed by others in a payment route. Transport protocols on top of the interledger protocol MAY take advantage of this capability to provide reliable payments, for example [UTP](../0006-universal-transport-protocol/) and [ATP](../0007-atomic-transport-protocol/). Some higher level protocols, such as [OTP](../0005-optimistic-transport-protocol/), may not use holds.
```

Comment:
Current wording (probably unintentionally) implies OTP is higher-level than UTP/ATP.